File: satire/testing/satireReal_compare.js

// This is a contant

const STARTING_X = 5;  
const STARTING_Y = 5;  

const TWO_PI = 6;  

var radiusValue = [5];

// This is a comment

function findRadius(centerX, centerY, radius, pointX, pointY) {  
	var multiX = centerX - pointX;
	var multiY = centerY - pointY;
	var multiXY = 0;
	multiXY = (multiX * multiX) + (multiY * multiY);
	if (multiXY >= (radius * radius)) {
		return 0;
	}
	return 1;
}

function areaOperator(findX, findY, loop) {   
	var valueOut = 0; 
	valueOut = (TWO_PI * loop) / ((findX * findX) + (findY * findY));
	return valueOut;
}

function partialFind(centerX, centerY, radius) {   
	const INCREMENTER = 1;  
	var loop = 0;
	var localRadius = [5];
	var localFloat = [5];
	while (loop < radius) {
		var findX = centerX - loop;
		var findY = centerY - loop;
		radiusValue[loop] = findRadius(centerX, centerY, radius, findX, findY);
		localFloat[loop] = areaOperator(findX, findY, loop);
		localRadius[loop] = radiusValue[loop] + INCREMENTER;
		console.log("Some command");    
		console.log(radiusValue[loop]);    
		console.log(localFloat[loop]);    
		loop += localRadius[loop];
	}
}

partialFind(STARTING_X, STARTING_Y, 5);

File: satire/testing/satireReal_compare.rb

#This is a contant  

$STARTING_X = 5  
$STARTING_Y = 5  
$INCREMENTER = 1  

$TWO_PI = 6  

$radiusValue = Array.new(5)

#This is a comment  

def findRadius(centerX, centerY, radius, pointX, pointY)      
	multiX = centerX - pointX    
	multiY = centerY - pointY    
	multiXY = 0    
	multiXY = (multiX * multiX) + (multiY * multiY) 
	if (multiXY >= (radius * radius))   
		return 0 
	end  
	return 1 
end  

def areaOperator(findX, findY, loop)      
	valueOut = 0    
	valueOut = ($TWO_PI * loop) / ((findX * findX) + (findY * findY)) 
	return valueOut 
end  

def partialFind(centerX, centerY, radius)      
	loop = 0    
	localRadius = Array.new(5)
	localFloat = Array.new(5)
	while (loop < radius)   
		findX = centerX - loop    
		findY = centerY - loop    
		$radiusValue[loop] = findRadius(centerX, centerY, radius, findX, findY) 
		localFloat[loop] = areaOperator(findX, findY, loop) 
		localRadius[loop] = $radiusValue[loop] + $INCREMENTER 
		puts "Some command"    
		puts $radiusValue[loop]      
		puts localFloat[loop]      
		loop += localRadius[loop] 
	end  
end  

partialFind($STARTING_X, $STARTING_Y, 5) 
  

